Source/GSXibLoader.m

```diff
@@ -923,6 +948,12 @@ - (void) awake: (NSArray *)rootObjects
   NSCustomObject *object;
   NSString *className;
 
+  if ([rootObjects count] == 0)
+  {
+    NSWarnMLog(@"no root objects!!!");
```

Comment:
Three exclamation points are maybe too much

Reply:
I am terribly sorry for that. As you can see the indentation is also wrong. I must have missed these lines when adding this code.

Reply:
> I am terribly sorry for that.

It's not like I haven't written or let through code like this, maybe even submitted it to GNUstep :D